composer/helpers/src/Hasher.php

```diff
@@ -4,18 +4,16 @@
 
 namespace Dependabot\Composer;
 
-use Composer\Factory;
+use Composer\Package\Locker;
 
 class Hasher
 {
     public static function getContentHash(array $args): ?string
     {
         [$workingDirectory] = $args;
 
-        $io = new ExceptionIO();
-        $composer = Factory::create($io, $workingDirectory . '/composer.json');
-        $locker = $composer->getLocker();
+        $config = $workingDirectory . '/composer.json';
 
-        return $locker->getContentHash(file_get_contents(Factory::getComposerFile()));
+        return Locker::getContentHash(file_get_contents($config));
```

Comment:
This method has been `static`, and still is, see

* https://github.com/composer/composer/blob/1.0.0/src/Composer/Package/Locker.php#L65-L102
* https://github.com/composer/composer/blob/1.9.0/src/Composer/Package/Locker.php#L65-L102